spec/features/article_admin_feature_spec.rb

```diff
@@ -40,11 +36,7 @@
 
   scenario "admin user deletes article" do
     @article = create(:article)
-    visit articles_path
-    click_link "New Article"
-    fill_in :article_title, with: @article.title
-    fill_in :article_body, with: @article.body
-    click_button "Submit"
+    visit article_path(@article)
     click_link "Delete Article"
     page.driver.browser.switch_to.alert.accept
```

Comment:
Favor using Capybara's built-in methods over diving into the `page.driver.browser` object. 

In this case, see `accept_alert`: http://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSession%3Aaccept_alert

Capybara has a method for pretty much everything you can dream of doing in the browser.